back/src/Guide/Api/Methods.hs

```diff
@@ -247,39 +247,41 @@ dbUpdate db x = liftIO $ do
 -- | Read something from the database.
 dbQuery :: (MonadIO m, EventState event ~ GlobalState, QueryEvent event)
         => DB -> event -> m (EventResult event)
```

Comment:
Perhaps it could be changed to `event -> Guider (EventType event)`? Then we can avoid passing `chDB` everywhere.

Reply:
:+1: